api-catalog-ui/frontend/src/components/Footer/Footer.jsx

```diff
@@ -19,7 +19,9 @@ export default class Footer extends Component {
                     </p>
                 </div>
                 <div className="right">
-                    <Link href="https://support.ca.com/us.html">CA Support</Link>
+                    <Link data-testid="link" href="https://support.ca.com/us.html">
```

Comment:
it seems that this link doesn't work anymore. We should also replace CA with Broadcom

Reply:
The footer is hidden since it will be displayed only for the enterprise product. Did you try to modify the `process.env.REACT_APP_CA_ENV` to true?
Yes I guess it should be changed to Broadcom now

Reply:
Yes you're right, I'll change it to Broadcom and replace with working link